### DISCUSSION TITLE
Problem opening loop since update

### DISCUSSION THREAD

**Discussion by AaronWNW** [mrkai77/Loop#455] [Q&A]
Hey, I did the update a week ago. Since then loop won't open (the circle won't open the loop setting work just fine). I tried resetting the trigger key but it didn't help. Am I the only one with the problem?

**Comment by SenpaiHunters** [comment]
Hey, do you have any logs, like crash logs from Apple? that can give us an idea of why's not working? And does the previous version, https://github.com/MrKai77/Loop/releases/download/1.0.0-beta.15/Loop.zip, have this issue?

**Reply by AaronWNW** [reply]
No it didn't crash. I also don't know how to get something like a core dump for loop.
The issue wasn't there with the old version (the .zip you mentioned). Once I quit the old version and start the new one it doesn't work anymore. 
Every time I open Loop it says that it needs some permissions in the settings. 
<img width="281" alt="Bildschirmfoto 2024-06-24 um 12 35 15" src="https://github.com/MrKai77/Loop/assets/124676511/723dfb7a-bb43-4076-9cd4-01ec85c7b506">

But im pretty sure that I gave all the permissions and if I click on "open settings" it shows that I already gave all the permissions. (Bedienungshilfen = Accessibility)

<img width="474" alt="Bildschirmfoto 2024-06-24 um 12 36 04" src="https://github.com/MrKai77/Loop/assets/124676511/7056564f-5489-48c9-9ef9-d551e2a17631">
<img width="555" alt="Bildschirmfoto 2024-06-24 um 12 37 57" src="https://github.com/MrKai77/Loop/assets/124676511/94cfc789-49de-4a55-aa6b-e14c4a0a219b">


Sorry for my English, im German.

**Reply by AaronWNW** [reply]
If you need anything else from me, im happy to help.

**Reply by mrkai77** [reply]
@AaronWNW try the instructions posted in #418

**Reply by AaronWNW** [reply]
It worked thx
